pkg/volume/util.go

```diff
@@ -193,6 +193,15 @@ func GenerateVolumeName(clusterName, pvName string, maxLength int) string {
 	return prefix + "-" + pvName
 }
 
+// Check if the path from the mounter is empty.
+func GetPath(mounter Mounter) (string, error) {
+	path := mounter.GetPath()
+	if path == "" {
+		return "", fmt.Errorf("Path is empty")
```

Comment:
can we please include the details of the driver and any other context about the Mounter, so if it happens again we will know who the bad actor is?